### DISCUSSION TITLE
The horizontal movement of a square is not smooth

### DISCUSSION THREAD

**Discussion by Cristian260121** [bonsai-rx/bonsai#1398] [General]
Dears:
I did a visual stimulus consisting of a square that changes its "x" position, then repeated for several "y" positions. For that, I used "Concat" as an example that appears in the Bonvision webpage. 

I am using two monitors of 120 Hz. However, sometimes, the movement of the square is not smooth. It sometimes stop shortly and then continue. 
Do you have some recommendations? Is it possible to improve the "x" movement with some change in the code, or is it necessary to change the monitors

Best regards
![image_1](https://github.com/bonsai-rx/bonsai/assets/130599729/4b99d784-1186-4daa-9727-d955f4d4d185)
![image_2](https://github.com/bonsai-rx/bonsai/assets/130599729/9c8dc999-c067-4093-b383-3aa8449e1013)
![image_3](https://github.com/bonsai-rx/bonsai/assets/130599729/cdedc072-dc06-474d-969a-8a4a56efd54d)

**Comment by bruno-f-cruz** [comment]
Can you include the workflow file? thanks!

**Reply by Cristian260121** [reply]
Yes, Thanks
[barrido_prueba2.zip](https://github.com/bonsai-rx/bonsai/files/11628852/barrido_prueba2.zip)

**Reply by bruno-f-cruz** [reply]
I don't see anything obviously wrong with the workflow. In fact, on my computer, it looks perfectly smooth. I believe your graphics pipeline just cant handle the frequencies you are trying to render stability resulting in display stutters. Can you try a different computer just to be sure?
